Sources/TensorFlow/Loss.swift

```diff
@@ -194,6 +194,14 @@ public func kullbackLeiblerDivergence<Scalar: TensorFlowFloatingPoint>(
     reduction(expected * log(expected / predicted))
 }
 
+/// Workaround for cross-module default parameter @differentiable functions.
+@differentiable
+public func _justTakesTheMean<Scalar: TensorFlowFloatingPoint>(
```

Comment:
Could you please rename this to just `mean`? If that's ambiguous, then `_mean`?
```suggestion
public func mean<Scalar: TensorFlowFloatingPoint>(
```